packages/application-config/schema.json

```diff
@@ -111,6 +111,9 @@
             },
             "teamId": {
               "type": "string"
+            },
+            "applicationId": {
+              "type": "string"
```

Comment:
We already have it in the `production` section. Do you think we need to have it duplicated here? It would probably be the same ID so I think we can rely on the `env.production.applicationId`.